docs/sdk_python/create_model.mdx

```diff
@@ -25,17 +25,32 @@ my_model = project.create_model (
 )
 ```
 
-Here is the syntax to create an OpenAI model:
+Please note that in the case of LLM models, the parameters are stored in `options`. Here is the syntax to create an OpenAI model:
 
 ```bash
-sentiment_classifier = project.create_model(
+sentiment_classifier = project.create_model (
```

Comment:
also the new syntax is :
`sentiment_classifier = project.models.create (`

Reply:
So it is also `project.models.get()` instead of `get_model()`.